src/main/java/com/ethanyidong/bunny/cmd/ListCommand.java

```diff
@@ -0,0 +1,19 @@
+package com.ethanyidong.bunny.cmd;
+
+import com.ethanyidong.bunny.BunnySession;
+
+import java.util.ArrayList;
+
+public class ListCommand extends ExecutableCommand {
+    public void execute(BunnySession bunny) {
```

Comment:
Consider avoiding deep nesting to improve readability. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting)